mainDoc.md

```diff
@@ -0,0 +1,139 @@
+# Getting started with NvChad
+
+Welcome to NvChad!
+
+Firstly, thank you for using this project! We appreciate each and every contributor & user of NvChad.
+
+![NvChad logo](https://user-images.githubusercontent.com/58810330/129740641-d3369cc0-19d6-4e2c-a4c5-eb5629da6d62.png "NvChad logo")
+
+## Install
+Installing is as easy as git cloning into your NeoVim configuration folder.
+
+If you already have a `~/.config/nvim` folder, move it with:
+```
+mv ~/.config/nvim ~/.config/Nvim.backup
+```
+**Download NvChad**:
+```
+git clone https://github.com/NvChad/NvChad.git ~/.config/nvim
+```
+Install all plugins:
+```
+:PackerSync
+```
+(note the **:** signifies this is a command to be run inside NeoVim)
+
+## Notable mappings
+
+| Key mapping  | Action | Note |
+| ------------- |:-------------:| |
+| `jk`          | <ESC> to normal mode | |
+| `<SHIFT> + t` | open a new buffer ||
+| `<SHIFT> + x` | close current buffer | this hides a NvChad terminal |
+| `<TAB>`       | cycle along the bufferline | `<SHIFT> + <TAB>` to reverse |
+| `<CNTL> + n`  | open NvimTree explorer | `<ENTER>` to select file |
+| `<SPCAE> + d + k` | view key mappings | |
+| `<SPACE> + /` | toggle commenting a line | |
+| `<SPACE> + f + f` | pick a file | Telescope picker | 
+| `<SPACE> + g + s` | git status | Telescope picker | 
+
+## Basic NvChad structure overview
+
+```
+project
+│   README.md
+│   init.lua 
+│
+└───lua
+│   │   chadrc.lua              -- the main configuration file for users
+|   |                              options, enable/disable plugins, keymaps
+|   |   default_config.lua      -- a provided defaults file (DO NOT MODIFY)
+|   |   mappings.lua            -- where the key-mappings are applied
+|   |   options.lua             -- where the options are applied
+|   |   packerInit.lua          -- tool to bootstrap Packer if not present
+|   |   pluginList.lua          -- the list of plugins for Packer
+|   |   utils.lua               -- various lua utilities functions
+|   |   theme.lua
+|   |   highlights.lua
+|   |   
+│   └───plugins
+│   |       autopairs.lua
+│   |       autosave.lua
+│   |       ...
+|   |       others.lua          -- small configurations for several plugins
+|   |       ...                 -- many other configurations
+|   |
+│   └───themes
+│   |   │   onedark.lua         -- the defalt theme
+│   |   │   gruvchad.lua
+|   |   |   ...                 -- other included themes
+│   │
+│   └───telescope
+│       └───_extensions
+|           |   themes.lua      -- custom Telescope picker for changing themes
+|           |   terms.lua       -- custom Telescope picker for terminals
+│   
+└───plugin                      -- managed by Packer
+    │   packer_compiled.lua     -- the compiled plugins
+```
+## How to customise NvChad
```

Comment:
I feel like it's redundant to say "NvChad" since everything under this org is NvChad related. IMO `## Customization` would look better.